plugin/mutate/source/dextool/plugin/mutate/backend/database/schema.d

```diff
@@ -433,6 +435,8 @@ struct MarkedMutantTbl {
     string rationale;
 
     ulong kind;
+
+    Language lang;
```

Comment:
It was purely added so that creation of MutationTextResult in report/plain.d (row 73) was more reasonable. I agree that the information is not needed for the purpose of displaying the table etc, but instead of doing it the "long way", it was a "nice-to-have". Is it still reasonable?